canopy/app/src/components/navigation/SideNav.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2019 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import 'canopy-design-system/src/themes/default';
+
+.side-nav {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  top: 0;
+  left: 0;
+  width: fit-content;
+  height: 100vh;
+  background: #ffffff;
```

Comment:
The sidebar blips before rendering the Nav Items. You could fix this with a min-width.

`min-width: ($padding-s * 3) + 2rem + 9rem;`

This isn't the most elegant solution because the nav-item styles are in a different file.

Really, just adding those Nav Item definitions to the same files as the SideNav classes would clean this up. And it would be totally fine to do so because NavItems are always a child of SideNav.

If you did that, you could have something much easier to keep in sync with a SCSS map
```
$nav-tab-spacings: (
  padding: $padding-s,
  column-gap: $padding-s,
  icon-width: 2rem,
  label-width: 9rem
);
$total-nav-width: map-get($nav-tab-spacings, 'padding') * 2 +
  map-get($nav-tab-spacings, 'column-gap') +
  map-get($nav-tab-spacings, 'icon-width') +
  map-get($nav-tab-spacings, 'label-width');

.side-nav {
  display: flex;
  flex-direction: column;
  position: relative;
  top: 0;
  left: 0;
  width: fit-content;
  height: 100vh;
  background: #ffffff;
  min-width: $total-nav-width;

  hr {
    border: 0;
    width: 100%;
    background-color: $background-light;
    height: 2px;
  }
}

.brand {
  display: flex;
  flex-direction: column;
  align-items: center;
  text-decoration: none;
  color: darken($color-canopy, 20%);
  padding: $padding-s;
  color: text-primary;

  h5 {
    margin: 0;
  }
}

.nav-tab {
  display: grid;
  grid-template-columns:
    [icon-start] map-get(
      $nav-tab-spacings,
      'icon-width'
    ) [icon-end label-start] map-get($nav-tab-spacings, 'label-width')
    [label-end];
  column-gap: map-get($nav-tab-spacings, 'column-gap');
  align-items: center;
  padding: map-get($nav-tab-spacings, 'padding');
  text-overflow: ellipsis;
  overflow: hidden;
  text-decoration: none;
  color: $text-primary;
  transition: all 0.2s linear;

  .icon {
    height: map-get($nav-tab-spacings, 'icon-width');
    width: map-get($nav-tab-spacings, 'icon-width');
  }
```

Reply:
Other than that, LGTM

Reply:
Nice tip! Updated